From 053204f4a243559644af2c47ae9895810685cb5a Mon Sep 17 00:00:00 2001
From: Dan Dennedy <dan@dennedy.org>
Date: Sun, 8 Jan 2017 10:22:33 -0800
Subject: [PATCH] Only use --std=c++11 CXXFLAG for g++.

The addition of --std=c++11 breaks build on my OS X 10.8 build agent,
which uses clang version "Apple LLVM version 8.0.0 (clang-800.0.42.1)"
---
 src/modules/qt/Makefile | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/src/modules/qt/Makefile b/src/modules/qt/Makefile
index e7b55ae..d88e519 100644
--- a/src/modules/qt/Makefile
+++ b/src/modules/qt/Makefile
@@ -39,7 +39,11 @@ ifneq ($(targetos), MinGW)
 endif
 endif
 
 CXXFLAGS := $(QTCXXFLAGS) $(CXXFLAGS) $(CFLAGS) $(EXIFCXXFLAGS) $(KDECXXFLAGS) -Wno-deprecated
+
+ifneq (, $(shell $(CXX) --version | grep -is gcc))
+    CXXFLAGS += --std=c++11
+endif
 
 LDFLAGS += $(QTLIBS) $(EXIFLIBS) $(KDELIBS)
 
